cli.js

```diff
@@ -67,7 +72,23 @@ process.stdin.on('keypress', (ch, key) => {
 		query.pop();
 	} else if (key.ctrl && key.name === 'u') {
 		query.length = 0;
-	} else if (key.name === 'return' && prevResult) {
+	} else if (key.ctrl && key.name === '0') {
+		npc.copy(prevResult);
+	} else if (key.ctrl && key.name === '1') {
```

Comment:
Too much repetition. Just do `key.ctrl && Number(key.name) >= 1 && Number(key.name) <= 7` and `prevResult[Number(key.name) - 1]`.